### PR TITLE
Feature/new frontend setup

### PR DESCRIPTION
**Pull request by liamjohnston**
FED initial setup. 
- Includes all starterkit files.
- Had to do some different stuff with static paths due to the directory structure of this repo not matching our own usual (cookiecutter) structure.
- After a bunch of trial and error on static paths, I have the CSS and JS bundles loading.
- Template generation for `base.html` and `500.html` happen in `templates_src` and get output to `templates/`. I guess I should do the same with 404. Or do we even need to do this? I don't even know. It seems to only exist for the static asset paths (css/js bundles).

Screenshot 2024-05-10 at 10 11 52 AM

### CODE REVIEW

File: cdhweb/blog/templates/blog/blogpost_detail.html [hunk 1]
```diff
@@ -2,7 +2,6 @@
 {% load wagtailcore_tags humanize %}
 {# simple blog post page; adapted in part from mezzanine blog post template #}
 
-{% block headattrs %} xmlns:article="http://ogp.me/ns/article#"{% endblock %}
```
Comment (liamjohnston) [inline cdhweb/blog/templates/blog/blogpost_detail.html:5]: no idea if needed/useful

File: cdhweb/settings/components/base.py [hunk 2]
```diff
@@ -33,7 +33,7 @@
 # Additional locations of static files
 # Always use forward slashes, even on Windows.
 # Don't forget to use absolute paths, not relative paths.
-STATICFILES_DIRS = [BASE_DIR / "sitemedia"]
+STATICFILES_DIRS = [BASE_DIR / "cdhweb/static"]
```
Comment (liamjohnston) [inline cdhweb/settings/components/base.py:36]: @haydngreatnews FYI

File: package.json [hunk 3]
```diff
@@ -1,22 +1,62 @@
 {
-  "private": true,
+  "cms": "django",
   "name": "cdh-web",
-  "description": "Django web application for the CDH Website",
-  "repository": "https://github.com/Princeton-CDH/cdh-web.git",
-  "author": "The Center for Digital Humanities at Princeton <cdhdevteam@princeton.edu>",
-  "license": "Apache-2.0",
```
Comment (liamjohnston) [inline package.json:7]: Do any of these need retaining?

File: templates/base.html
Comment (liamjohnston) [inline templates/base.html:1]: if we're going to make this a generated file, should it be version controlled or ignored?

File: cdhweb/templates_src/base.html [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+    {% endcomment %}
+    {% block extra_css %}
+      {# Needed for specific cases #}
+    {% endblock %}
+
+  </head>
+  <body class="{% block body_class %}{% endblock %}">
+    
+    {% block content %}{% endblock %}
+    {% block main %}{% endblock %}
```
Comment (liamjohnston) [inline cdhweb/templates_src/base.html:30]: Not sure whether both are needed

Comment (haydngreatnews) [reply cdhweb/templates_src/base.html:30]: Yeah, seems an arbitrary choice. I'd certainly be inclined to pick just one, for consistency

**Review by haydngreatnews — COMMENTED** [review]
backend bits and tags look sensible to me (as does the general vibe)

**Review by haydngreatnews — APPROVED** [review]

**Pull request merged**